zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/processinstance/ProcessInstanceMigrationMigrateProcessor.java

```diff
@@ -285,6 +286,28 @@ private void tryMigrateElementInstance(
                         .setBpmnProcessId(targetProcessDefinition.getBpmnProcessId())
                         .setTenantId(elementInstance.getValue().getTenantId())));
 
+    if (ProcessInstanceIntent.ELEMENT_ACTIVATING != elementInstance.getState()) {
+      // element in ACTIVATING state should not be subscribed into new subscription during migration
```

Comment:
🔧 This comment doesn't explain the "why". Let's mention the ACTIVATING element will not have any message subscriptions created yet, so we shouldn't create one in the target process. The subscription will be created when the incident is resolved in the target element.